src/test/kotlin/io/bazel/kotlin/builder/tasks/jvm/KotlinWorkerTest.kt

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2020 The Bazel Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package io.bazel.kotlin.builder.tasks.jvm
+
+import com.google.common.truth.Truth.assertThat
+import io.bazel.kotlin.builder.KotlinJvmTestBuilder
+import io.bazel.kotlin.builder.tasks.BazelWorker.Companion.OK
+import io.bazel.kotlin.builder.tasks.InvocationWorker
+import io.bazel.kotlin.builder.tasks.KotlinBuilder.Companion.JavaBuilderFlags
+import io.bazel.kotlin.builder.tasks.KotlinBuilder.Companion.KotlinBuilderFlags
+import io.bazel.kotlin.model.CompilationTaskInfo
+import io.bazel.kotlin.model.Platform
+import io.bazel.kotlin.model.RuleKind
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import java.io.ByteArrayOutputStream
+import java.io.IOException
+import java.io.OutputStream
+import java.io.PrintStream
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+import java.nio.file.Path
+import java.util.zip.ZipEntry
+import java.util.zip.ZipFile
+import kotlin.streams.toList
+
+@RunWith(JUnit4::class)
+class KotlinWorkerTest {
+
+  private val wrkDir = Files.createTempDirectory("KotlinBuilderEnvironmentTest")
```

Comment:
Of course, if we WERE injecting FileSystem, then we could be using JIMFS. :)

Reply:
Well. Um. Not really. The compiler only consumes strings, not paths.